plutus-tx/src/PlutusTx/Lift.hs

```diff
@@ -63,6 +65,8 @@ safeLift
        , PLC.Typecheckable uni fun
        , PrettyUni uni, Pretty fun
        , Default (PLC.CostingPart uni fun)
+       , Default (PIR.BuiltinsInfo uni fun)
+       , Default (PIR.RewriteRules uni fun)
```

Comment:
Constraints in Plutus Tx are completely out of order.

Reply:
By order you mean, i should stick to a certain e.g. alphabetical order accoss the file?

Reply:
Sorry, I mean in the sense of "If something someone says or does is out of order, it is unpleasant or not suitable and it is likely to upset or offend people". They're way too bloated, but I don't know if we can have sensible type synonyms there.